### DISCUSSION TITLE
Welcome to Coqui STT Discussions!

### DISCUSSION THREAD

**Discussion by JRMeyer** [coqui-ai/STT#1790] [General]
# :wave: Welcome to :frog:STT Discussions!

We’re using Discussions as a place to connect with members of our community and talk about :frog:STT

Feel free to ask whatever questions you have related to the project, the code, the models, or your use-case.

:heavy_check_mark: Share ideas :bulb: 
:heavy_check_mark: Engage with other community members :people_holding_hands:
:heavy_check_mark: Ask questions :question: and get answers :+1:

Please help us make this a healthy, welcoming community

:heavy_check_mark: Welcome others, be polite, and have an open mind :)

Remember that this is a community we build together :mechanical_arm: 

Feel free to comment below with an introduction of yourself!

**Comment by Eren94** [comment]
Hi guys,

I will gladly introduce myself. I am Eren and started working on a research project in Germany concerning smart speakers about a year ago. The whole topic got me fascinated and I am working on my own voice assistant now. Started out using Mozilla DeepSpeech, now will be switching to Coqui :)
I hope I will learn a lot from this community and be able to contribute to it aswell and help others out. Always feel free to message me :)
